### PR TITLE
fix: focus on textarea

### PR DESCRIPTION
**Pull request by harmanbatheja15**
Fixes issue #11378 
Also fixed **border-color** change on hover on textarea.

Screenshot 2023-09-26 at 2 39 09 AM

### CODE REVIEW

File: packages/ui/components/editor/Editor.tsx [hunk 1]
```diff
@@ -68,7 +68,7 @@ export const Editor = (props: TextEditorProps) => {
   return (
     <div className="editor rounded-md">
       <LexicalComposer initialConfig={{ ...editorConfig, editable }}>
-        <div className="editor-container rounded-md p-0">
+        <div className="editor-container rounded-md p-0 hover:border-emphasis">
```
Comment (sean-brydon) [inline packages/ui/components/editor/Editor.tsx:71]:
```suggestion
        <div className="editor-container hover:border-emphasis focus-within:ring-brand-default rounded-md p-0 focus-within:ring-2">

File: packages/ui/components/editor/stylesEditor.css [hunk 2]
```diff
@@ -27,6 +27,10 @@
   padding: 1px;
 }
 
+.editor-container:focus-within {
+  box-shadow: var(--tw-ring-inset) 0 0 0 calc(2px + var(--tw-ring-offset-width)) #fff;
+}
+
```
Comment (sean-brydon) [inline packages/ui/components/editor/stylesEditor.css:33]:
```suggestion

```

**Review by sean-brydon — DISMISSED** [review]
I think we can do the same just using focus within and using the same styles as our input component.

File: packages/ui/components/editor/Editor.tsx [hunk 3]
```diff
@@ -68,7 +68,8 @@ export const Editor = (props: TextEditorProps) => {
   return (
     <div className="editor rounded-md">
       <LexicalComposer initialConfig={{ ...editorConfig, editable }}>
-        <div className="editor-container rounded-md p-0">
+        <div className="editor-container hover:border-emphasis focus-within:ring-brand-default rounded-md p-0 focus-within:ring-2">
+
```
Comment (PeerRich) [inline packages/ui/components/editor/Editor.tsx:72]:
```suggestion
        <div className="editor-container hover:border-emphasis focus-within:ring-brand-default rounded-md p-0 focus-within:ring-2">
```

**Review by PeerRich — APPROVED** [review]

**Pull request merged**